Midterm2020/Midterm2020/bin/Debug/netcoreapp3.1/Midterm2020.runtimeconfig.dev.json

```diff
@@ -1,8 +1,8 @@
 {
   "runtimeOptions": {
     "additionalProbingPaths": [
-      "C:\\Users\\Mike\\.dotnet\\store\\|arch|\\|tfm|",
-      "C:\\Users\\Mike\\.nuget\\packages"
+      "C:\\Users\\darsh\\.dotnet\\store\\|arch|\\|tfm|",
```

Comment:
Is there a way to make this dynamic?

Reply:
string path = Path.Combine(Environment.CurrentDirectory, @"\Midterm2020\Midterm2020\MyText.txt");
Here we go.